kernel/src/main/scala/org/typelevel/sbt/TypelevelKernelPlugin.scala

```diff
@@ -0,0 +1,27 @@
+package org.typelevel.sbt
+
+import sbt._, Keys._
+import sbt.plugins.JvmPlugin
+
+object TypelevelKernelPlugin extends AutoPlugin {
+
+  override def requires = JvmPlugin
+  override def trigger = allRequirements
+
+  object autoImport {
+    lazy val tlIsScala3 = settingKey[Boolean]("True if building with Scala 3")
+    def replaceCommandAlias(name: String, contents: String): Seq[Setting[State => State]] =
```

Comment:
Does this deserve a `tl` prefix since it's auto-imported?

Reply:
auto-imports definitely need a prefix. The similar `isDotty` setting in sbt-spiewak is the reason http4s is not in the Scala 3 Community Build:
- https://github.com/djspiewak/sbt-spiewak/issues/83